cps/editbooks.py

```diff
@@ -328,7 +328,7 @@ def media():
         abort(404)
 
     def request_media_download(media_url, original_url):
-        task_message = N_("learning media from %(media_url)s", media_url=media_url)
+        task_message = N_("fetching %(media_url)s...", media_url=media_url)
```

Comment:
@deldesir if it's more accurate, can we use the word "downloading" instead of the word "fetching" ?

Maybe I'm misunderstanding.  But I personally find the word "fetching" very ambiguous:

- As it does not tell me where it's coming from.
- As it does not tell me where it's going.

Reply:
The task name is "Download". The message will appear to user as " Download: downloading [URL]...".